knowledge-base/codacy/codacy-analysis-cli-action/action-security.yml

```diff
@@ -0,0 +1,18 @@
+name: 'Codacy Analysis CLI'
+outbound-endpoints:
+  - fqdn: raw.githubusercontent.com
```

Comment:
@arjundashrath this is downloading Codacy CLI. can you please change it to `to download Codacy CLI`?
https://github.com/codacy/codacy-analysis-cli-action/blob/9f09141804faa25deeb6d09264f7e894f6c50ef9/action.yml#L228